### ISSUE TITLE
Allow to add fallback if no label match occurs

### ISSUE THREAD

**Issue by jbinda** [IvanFon/super-labeler-action#13]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Abe able to pass fallback if the is no match for label when opening PR/issue (default label can be `need_triage` or something like that

**Comment by jayu** [comment]
Hi folks, how about the following syntax:
```
{
   labels:{},
   pr : {},
   pr_fallback : "", // name of label to fallback, eg. needs triage
   issue: {},
   issue_fallback : "" // name of label to fallback, eg. needs triage
}
```
I would suggest to not set any fallback label if not explicitly specified.

**Comment by jbinda** [comment]
> pr_fallback : "", // name of label to fallback, eg. needs triage

Do you think it should be one label or an Array ?

Also I wonder if apart of config we do not want some flag in yml to disable fallback as well. I think about the use case when I do not want to change config but e.g temporary disable fallback

**Comment by jayu** [comment]
Array seems to be more reasonable and future proof, you are right.
Regarding the second sentence, I'm not sure if it is really needed

**Comment by IvanFon** [comment]
> Also I wonder if apart of config we do not want some flag in yml to disable fallback as well. I think about the use case when I do not want to change config but e.g temporary disable fallback

I don’t think this is necessary - it doesn’t seem like a very common scenario, and since the config is checked into the repo it’s easy to revert it after a temporary change. I think having configuration specified in multiple places would increase complexity too much.

> Array seems to be more reasonable and future proof, you are right.

Agreed 👍

**Comment by jayu** [comment]
I'm working on this :)
